source/renderer/vs_modelloader.cpp

```diff
@@ -1,6 +1,7 @@
 #include <assimp/scene.h>
 #include <assimp/postprocess.h>
 #include <assimp/Importer.hpp>
+#include <iostream>
```

Comment:
???

Reply:
Actually needed in this class because it does not use our logger yet